src/main/java/cmsc495/ui/Page_BrowseRecipe.java

```diff
@@ -114,7 +119,7 @@ public void actionPerformed(ActionEvent e) {
         
         // set the panel to the main page
         SimpleGui gui = (SimpleGui)SwingUtilities.getRoot(button);
-        gui.setCurrentPage(new Page_DisplayRecipe(recipe));
+        //gui.setCurrentPage(new Page_DisplayRecipe(recipe));
```

Comment:
@ahowell82 When I uncomment link 122.  I receive, "The constructor Page_DisplayRecipe(Recipe_Hold) is undefined."  Double checking my merge...